testing/acceptance_test.py

```diff
@@ -541,6 +542,28 @@ def test_foo(data):
         res = pytester.runpytest(p)
         res.assert_outcomes(passed=3)
 
+    @pytest.mark.filterwarnings("ignore:'encoding' argument not specified")
+    def test_command_line_args_from_file(
+        self, pytester: Pytester, tmp_path: Path
+    ) -> None:
+        pytester.makepyfile(
+            test_file="""
+            import pytest
+
+            class TestClass:
+                @pytest.mark.parametrize("a", ["x","y"])
+                def test_func(self, a):
+                    pass
+            """
+        )
+        tests = [
+            "test_file.py::TestClass::test_func[x]",
+            "test_file.py::TestClass::test_func[y]",
```

Comment:
Let's add some flag to ensure it remains working:

```suggestion
            "test_file.py::TestClass::test_func[y]",
            "-q",
```